packages/i18n/src/fi.ts

```diff
@@ -0,0 +1,83 @@
+    maxGraphemes:     (issue) => `Virheellinen grafeemimäärä: Odotettiin ${issue.expected}, mutta saatiin ${issue.received}`,
+    maxLength:        (issue) => `Virheellinen pituus: Odotettiin ${issue.expected}, mutta saatiin ${issue.received}`,
+    maxSize:          (issue) => `Virheellinen koko: Odotettiin ${issue.expected}, mutta saatiin ${issue.received}`,
+    maxValue:         (issue) => `Virheellinen arvo: Odotettiin ${issue.expected}, mutta saatiin ${issue.received}`,
+    maxWords:         (issue) => `Virheellinen sanamäärä: Odotettiin ${issue.expected}, mutta saatiin ${issue.received}`,
+    mimeType:         (issue) => `Virheellinen MIME-tyyppi: Odotettiin ${issue.expected}, mutta saatiin ${issue.received}`,
+    minBytes:         (issue) => `Virheellinen tavumäärä: Odotettiin ${issue.expected}, mutta saatiin ${issue.received}`,
+    minGraphemes:     (issue) => `Virheellinen grafeemimäärä: Odotettiin ${issue.expected}, mutta saatiin ${issue.received}`,
+    minLength:        (issue) => `Virheellinen pituus: Odotettiin ${issue.expected}, mutta saatiin ${issue.received}`,
+    minSize:          (issue) => `Virheellinen koko: Odotettiin ${issue.expected}, mutta saatiin ${issue.received}`,
+    minValue:         (issue) => `Virheellinen arvo: Odotettiin ${issue.expected}, mutta saatiin ${issue.received}`,
+    minWords:         (issue) => `Virheellinen sanamäärä: Odotettiin ${issue.expected}, mutta saatiin ${issue.received}`,
+    multipleOf:       (issue) => `Virheellinen monikerta: Odotettiin ${issue.expected}, mutta saatiin ${issue.received}`,
+    nanoid:           (issue) => `Virheellinen Nano ID: Saatiin ${issue.received}`,
+    nonEmpty:         (issue) => `Virheellinen pituus: Odotettiin ${issue.expected}, mutta saatiin ${issue.received}`,
+    notBytes:         (issue) => `Virheellinen tavumäärä: Odotettiin ${issue.expected}, mutta saatiin ${issue.received}`,
+    notGraphemes:     (issue) => `Virheellinen grafeemimäärä: Odotettiin ${issue.expected}, mutta saatiin ${issue.received}`,
+    notLength:        (issue) => `Virheellinen pituus: Odotettiin ${issue.expected}, mutta saatiin ${issue.received}`,
+    notSize:          (issue) => `Virheellinen koko: Odotettiin ${issue.expected}, mutta saatiin ${issue.received}`,
+    notValue:         (issue) => `Virheellinen arvo: Odotettiin ${issue.expected}, mutta saatiin ${issue.received}`,
+    notWords:         (issue) => `Virheelliset sanat: Odotettiin ${issue.expected}, mutta saatiin ${issue.received}`,
+    octal:            (issue) => `Virheellinen oktaaliluku: Saatiin ${issue.received}`,
+    partialCheck:     (issue) => `Virheellinen syöte: Saatiin ${issue.received}`,
+    regex:            (issue) => `Virheellinen muoto: Odotettiin ${issue.expected}, mutta saatiin ${issue.received}`,
+    safeInteger:      (issue) => `Virheellinen turvallinen kokonaisluku: Saatiin ${issue.received}`,
```

Comment:
Again, I'm hard-pressed to find an end-user to whom "turvallinen kokonaisluku" makes sense.